### PR TITLE
[update] changes regarding access_token removal [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by Catografix**
Docs to be included:
https://auth0.com/docs/migrations
https://auth0.com/docs/tokens/idp
https://auth0.com/docs/what-to-do-once-the-user-is-logged-in/calling-an-external-idp-api
https://auth0.com/docs/quickstart/native/xamarin
https://auth0.com/docs/user-profile/normalized
https://auth0.com/docs/quickstart/native/windows-uwp-javascript

### CODE REVIEW

File: articles/user-profile/normalized.md [hunk 1]
```diff
@@ -1,56 +1,62 @@
...
 
 The `identities` array contains the following attributes:
 
-* `access_token`: if the identity provider implements OAuth2, you will find the `access_token` that can be used to call the provider API and obtain more information about the user;
-* `access_token_secret`: **currently only for Twitter**. If the identity provider is OAuth 1.0a, an  `access_token_secret` property will be present and can be used to call the provider API and obtain more information from the user;
-* `connection`: the name of the connection;
-* `expires_in`: the length of time before the token expires;
-* `isSocial`: indicates if the provider is a Social provider;
-* `provider`: the provider of the connection;
+* `access_token_secret`: If the identity provider is using OAuth 1.0a, an `access_token_secret` property will be present that can be used to call the provider API and obtain more information about the user (currently only for Twitter).
```
Comment (Catografix) [inline articles/user-profile/normalized.md]: Is the twitter access_token_secret also being removed? Seems like a similar vulnerability.

Comment (mcastany) [inline articles/user-profile/normalized.md]: Yes, we remove this prop too. Also the `expires_in`

Comment (Catografix) [inline articles/user-profile/normalized.md]: Oh so I need to go back and remove the `expires_in` from all the docs

File: articles/tokens/idp.md [hunk 2]
```diff
@@ -1,49 +1,79 @@
...
+### 1. Get the user id from the user profile
 
-Identity Provider access tokens can be obtained in one of two ways.  First, if a user authenticates to a social Identity Provider, such as Facebook, an Identity Provider access token for that social Identity Provider will be returned in the first element of the `identities` array within the user profile object returned by Auth0.
+First, you will need to retrieve the `user_id` property from the user profile after successful authentication.
 
-For example, if authentication is invoked via a call to the Lock widget’s .show method, the following code would be used to put the Identity Provider access token into a variable called `identityProviderAccessToken`. .
+For example, if you are using [Lock](/libraries/lock), you can get the `user_id` from the returned profile with this code:
 
-```js
+```
```
Comment (Catografix) [inline articles/tokens/idp.md]: This is probably completely wrong, but this is to show the type of sample code required here.

File: articles/tokens/idp.md [hunk 3]
```diff
@@ -1,49 +1,79 @@
...
+  headers: { authorization: 'Bearer YOUR_API_TOKEN' } };
+
+request(options, function (error, response, user) {
+  if (error) throw new Error(error);
 
-## How to control contents of Identity Provider access tokens
+  console.log(user.identities[0].access_token);
+});
+```
+**NOTE:** The `YOUR_API_TOKEN` must contain the `read:user_idp_tokens` scope.
```
Comment (Catografix) [inline articles/tokens/idp.md:53]: We are not showing how to get that auth0 api token to make the call. is there a way to generate the token programmatically?

File: articles/tokens/idp.md [hunk 4]
```diff
@@ -1,49 +1,79 @@
...
 lock.show({
   responseType: 'token',
-  authParams: {
-     scope: 'openid name email'
-  }
-}, function(err, profile, token) {
-    identityProviderAccessToken = profile.identities[0].access_token;
+}, 
+  function(err, profile, token) {
+    user-id = profile.user_id;
```
Comment (Catografix) [inline articles/tokens/idp.md:28]: This is probably completely wrong, but this is to show the type of sample code required here.

File: articles/what-to-do-once-the-user-is-logged-in/calling-an-external-idp-api.md [hunk 5]
```diff
@@ -1,55 +1,21 @@
...
+
+Identity Providers commonly expose their own API that you can call with the access token granted to each user upon successful authentication with their service.
+
+Information on how to obtain an IdP access tokenis detailed at: [Identity Provider Access Tokens](/tokens/idp).
+
+Once you have the access token from the IdP for the user, you can call the IdP API.
+
+For example, to return a list of contacts for the user from Google:
+  
+`getGoogleContactsWithToken(googleToken);`
```
Comment (Catografix) [inline articles/what-to-do-once-the-user-is-logged-in/calling-an-external-idp-api.md]: Is this enough info, or should we show a function?

File: articles/tokens/idp.md (hunk 3 above)
Comment (mcastany) [inline articles/tokens/idp.md:53]: Let's include a note saying: `For information on how to get an access token you can check this link` (or something similar :) )

https://auth0.com/docs/api-auth/asking-for-access-tokens .

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by Catografix** [comment]
Superceded by #1718

**Pull request closed without merging**